hrm-domain/hrm-core/contact-job/client-sqs.ts

```diff
@@ -72,3 +79,18 @@ export const publishToContactJobs = async (params: PublishToContactJobsTopicPara
     console.error('Error trying to send message to SQS queue', err);
   }
 };
+
+export const scrubCompletedContactJobsFromQueue = async (
```

Comment:
This is a confusing name, the function actually posts a 'scrub transcript' job onto the queue. The equivalent function for retrieve transcripts is `publishRetrieveContactTranscript`, you should follow the same naming convention for this

Reply:
@stephenhand -  Feedback has been implemented and update pushed.